pkg/runtime/runtime.go

```diff
@@ -101,11 +101,11 @@ func (r *Runtime) Start() error {
 func (r *Runtime) isReady() bool {
 	// if at least a provider can watch for flags changes, we are ready.
 	for _, p := range r.SyncImpl {
-		if p.IsReady() {
-			return true
+		if !p.IsReady() {
```

Comment:
Sometimes I wish go had a array reducer. I'm sorry if that's too JS of me :sweat_smile: